src/executables/util_admin.c

```diff
@@ -895,7 +895,7 @@ static GETOPT_LONG ua_Flashback_Option[] = {
 
 static UTIL_ARG_MAP ua_Memmon_Option_Map[] = {
   {OPTION_STRING_TABLE, {0}, {0}},
-  {MEMMON_MODULE_S, {ARG_STRING}, {0}},
+  {MEMMON_MODULE_S, {ARG_INTEGER}, {(void *) -2}},
```

Comment:
```suggestion
  {MEMMON_MODULE_S, {ARG_INTEGER}, {(void *) -1}},
```
Could you tell me why it is -2?

Reply:
because -1 is for "-m all" case. So we have to avoid default value with -1.

Reply:
Ah, ok. All is not the default.